CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangelog.com/en/1.0.0/) from version 0.9 onwards.
 
 ## 0.17
+- **Implemented trend reversal candles**
```

Comment:
Please put these change logs under their appropriate 0.17 sub-section

Reply:
I've updated the changelog, does this version look fine to you?